easybuild/easyconfigs/e/ETSF_IO/ETSF_IO-1.0.4-intel-2015b.eb

```diff
@@ -0,0 +1,37 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2013-2014 The Cyprus Institute
+# Authors:: Thekla Loizou <t.loizou@cyi.ac.cy>
+# License:: MIT/GPL
+#
+##
+easyblock = 'ConfigureMake'
+
+name = 'ETSF_IO'
+version = '1.0.4'
+
+homepage = 'http://www.etsf.eu/resources/software/libraries_and_tools'
+description = """A library of F90 routines to read/write the ETSF file 
+format has been written. It is called ETSF_IO and available under LGPL. """
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+source_urls = ['http://www.etsf.eu/system/files']
+sources = [SOURCELOWER_TAR_GZ]
+
+configopts = "--with-netcdf-prefix=$EBROOTNETCDF "
+configopts += "--with-netcdf-libs='-L$EBROOTNETCDF/lib -lnetcdf -lnetcdff' "
+configopts += " --with-netcdf-incs='-I$EBROOTNETCDF/include'"
```

Comment:
ok, but then you're missing `-L$EBROOTNETCDFMINUSFORTRAN/lib`?

and maybe analogue for `include`